mono/metadata/w32file-unix.c

```diff
@@ -351,21 +357,26 @@ _wapi_chmod (const gchar *pathname, mode_t mode)
 			return -1;
 		}
 
+#if defined(HAVE_CHMOD)
```

Comment:
Whitespace. And chmod is in posix. Ifdef should not be needed & is therefore harmful. Some known system lacks it?